src/main/java/uk/me/desert_island/rer/rei_stuff/LootDisplay.java

```diff
@@ -228,7 +225,7 @@ public void munchLootCondition(JsonElement conditionElement, List<LootOutput> ou
                     outputs.addAll(munchLootSupplierJson(json));
                 break;
             case "minecraft:tag":
-                Tag<Item> tag = ItemTags.getTagGroup().getTag(new Identifier(object.get("name").getAsString()));
+                Tag<Item> tag = null;// TODO ItemTags.getTagGroup().getTag(new Identifier(object.get("name").getAsString()));
```

Comment:
I am unsure how to fix this. ``ItemTags#getTagGroup`` has been removed in 1.18.2 and I cannot seem to find a replacement

Reply:
Funny enough, [an article](https://gist.github.com/SizableShrimp/ddcbe9a9862cc4a0f526c42ae49b2c1d) on the Forge side seem to be helpful.

Anyway, I think you may want to look into the `Registry` class. It has a `streamTags` method but I don't know if that's what you are finding.

Reply:
Thanks for the article! However, I still don't know exactly how to fix this, sadly. Maybe I will read through the internals to see what I could do instead

Reply:
Okay, I got it!
The `streamTagsAndEntries` method did the trick. Together with some converting and mapping streams, it is finally working again!
Now, this PR is complete, thank you very much! :)